tests/system/general/test_08_Primitives3D.py

```diff
@@ -475,21 +475,36 @@ def test_21_sweep_along_path(self):
         udp2 = [5, 0, 0]
         udp3 = [5, 5, 0]
         arrofpos = [udp1, udp2, udp3]
-        path = self.aedtapp.modeler.create_polyline(arrofpos, name="poly_vector")
-        my_name = path.name
-        assert my_name in self.aedtapp.modeler.line_names
-        assert my_name in self.aedtapp.modeler.model_objects
-        assert my_name in self.aedtapp.modeler.object_names
-        assert isinstance(self.aedtapp.modeler.get_vertices_of_line(my_name), list)
-        rect = self.aedtapp.modeler.create_rectangle(self.aedtapp.PLANE.YZ, [0, -2, -2], [4, 3], name="rect_1")
-        swept = self.aedtapp.modeler.sweep_along_path(rect, path)
-        assert swept
-        assert rect.name in self.aedtapp.modeler.solid_names
+
+        path1 = self.aedtapp.modeler.create_polyline(arrofpos, name="poly_vector1")
+        path2 = self.aedtapp.modeler.create_polyline(arrofpos, name="poly_vector2")
+
+        paths = [path1.name, path2.name]
+        for item in paths:
+            assert item in self.aedtapp.modeler.line_names
+            assert item in self.aedtapp.modeler.model_objects
+            assert item in self.aedtapp.modeler.object_names
+            assert isinstance(self.aedtapp.modeler.get_vertices_of_line(item), list)
```

Comment:
@DaveTwyman for me this can be deleted, it's out of the scope of the test.

Reply:
Sounds good to me, they are just extensions from the existing unit test so I guess the scope has changed from when this test was originally written or they were always out of scope. 
Either way I've just deleted these. Thanks